src/runtime/recipe/handle.ts

```diff
@@ -170,11 +170,7 @@ export class Handle implements Comparable<Handle> {
     this._storageKey = storage.storageKey;
 
     if (storage.claims) {
```

Comment:
You can delete the if-statement, and just assign directly: `this.claims = storage.claims;` (the undefined will copy across correctly)

Reply:
Done.